camel/typing.py

```diff
@@ -19,6 +19,7 @@ class RoleType(Enum):
     USER = "user"
     CRITIC = "critic"
     EMBODIMENT = "embodiment"
+    ROLE_DESCRIPTION = "role_description"
```

Comment:
How can a role type be of value "role description". I do not get the concept.

Reply:
Considering your comment, I added a new type `DescriptionType`. in this type, it is available to define kinds of description types (like, skills, competences, duties). At present, I only defined a default description type:
```
# camel\typing.py
class DescriptionType(Enum):
    DEFAULT = "default_description"
```

Reply:
@lightaime can you take a look? We need someone smart to understand what is going on here.

Reply:
Please wait, I will modify the structure with this type tonight. And then, I will explain the changes I have made here. Thanks for your patience.